neo4j/config.go

```diff
@@ -77,6 +79,11 @@ func validateAndNormaliseConfig(config *Config) error {
 		config.ConnectionAcquisitionTimeout = -1
 	}
 
+	// Connection Liveness Check Timeout
+	if config.ConnectionLivenessCheckTimeout < 0 {
+		return &UsageError{Message: "Connection liveness check timeout cannot be smaller than 0"}
+	}
```

Comment:
Consider adding a unit test for this case (there already is a `TestValidateAndNormaliseConfig` that you can extend).

Reply:
Yep agree 👍